src/Features/Core/Portable/GenerateMember/GenerateConstructor/AbstractGenerateConstructorService.Editor.cs

```diff
@@ -1,449 +0,0 @@
-                    _state.TypeToGenerateIn,
-                    members,
-                    new CodeGenerationOptions(_state.Token.GetLocation()),
-                    _cancellationToken)
-                    .ConfigureAwait(false);
-
-                return (result, fields.Length > 0);
-            }
-
-            private async Task<(Document, bool addedFields)> GenerateFieldDelegatingConstructorAsync(NamingRule fieldNamingRule, NamingRule parameterNamingRule)
-            {
-                var arguments = _state.Arguments;
-                var parameterTypes = _state.ParameterTypes;
-
-                var typeParametersNames = _state.TypeToGenerateIn.GetAllTypeParameters().Select(t => t.Name).ToImmutableArray();
-                var parameterNames = GetParameterNames(arguments, typeParametersNames, parameterNamingRule);
-
-                GetParameters(arguments, _state.AttributeArguments, parameterTypes, parameterNames, fieldNamingRule, parameterNamingRule,
-                    out var parameterToExistingFieldMap, out var parameterToNewFieldMap, out var parameters);
-
-                var provider = _document.Project.Solution.Workspace.Services.GetLanguageServices(_state.TypeToGenerateIn.Language);
-                var syntaxFactory = provider.GetService<SyntaxGenerator>();
-                var codeGenerationService = provider.GetService<ICodeGenerationService>();
-
-                var syntaxTree = _document.SyntaxTree;
-                var (fields, constructor) = syntaxFactory.CreateFieldDelegatingConstructor(
-                    _document.SemanticModel,
-                    _state.TypeToGenerateIn.Name,
-                    _state.TypeToGenerateIn, parameters,
-                    parameterToExistingFieldMap, parameterToNewFieldMap,
-                    addNullChecks: false, preferThrowExpression: false);
-
-                var result = await codeGenerationService.AddMembersAsync(
-                    _document.Project.Solution,
-                    _state.TypeToGenerateIn,
-                    fields.Concat(constructor),
-                    new CodeGenerationOptions(_state.Token.GetLocation()),
-                    _cancellationToken)
-                    .ConfigureAwait(false);
-
-                return (result, fields.Length > 0);
-            }
-
-            private ImmutableArray<ParameterName> GetParameterNames(
-                ImmutableArray<TArgumentSyntax> arguments, ImmutableArray<string> typeParametersNames, NamingRule parameterNamingRule)
-            {
-                return _state.AttributeArguments != null
-                    ? _service.GenerateParameterNames(_document.SemanticModel, _state.AttributeArguments, typeParametersNames, parameterNamingRule, _cancellationToken)
-                    : _service.GenerateParameterNames(_document.SemanticModel, arguments, typeParametersNames, parameterNamingRule, _cancellationToken);
-            }
-
-            private void GetParameters(
```

Comment:
this code, and below was moved.  in general the overall logic stayed the same, however:

1. we pass less values (since they became state variables).
2. it was updated to support generate fields, not just properties.